jenkins_node_scanner.py

```diff
@@ -224,6 +216,14 @@ def main():
     while True:
         with ignore_exceptions_except_exit():
             with GLOBAL_EXCEPTIONS.count_exceptions():
+                logging.info('Connecting to Jenkins master at %s', args.url)
```

Comment:
How much spam is this producing then?

Reply:
Not much, but I guess it should be a debug line.